Lib/shutil.py

```diff
@@ -1582,3 +1580,14 @@ def which(cmd, mode=os.F_OK | os.X_OK, path=None):
                 if _access_check(name, mode):
                     return name
     return None
+
+def __getattr__(name):
+    if name == "ExecError":
+        import warnings
+        warnings._deprecated(
+            "shutil.ExecError",
+            f"{warnings._DEPRECATED_MSG}; use RuntimeError instead",
```

Comment:
Not sure that "use RuntimeError instead" makes sense as advice.

```suggestion
            f"{warnings._DEPRECATED_MSG}; it "
            "isn't raised by any shutil function.",
```